src/sgxlkl.edl

```diff
@@ -1,6 +1,11 @@
 // SGX-LKL-OE enclave/host interface
 
 enclave {
+    from "openenclave/edl/attestation.edl" import *;
```

Comment:
Please can you add a comment explaining what we get from each of these?  It looks as if this is importing more than we need (for example, we have our own ocall for CPUID emulation, which we are planning on removing because it's insecure and replacing with a table of CPUID leaf nodes that are valid and including that in the attestation).

Reply:
cpu.edl and debug.edl are currently always required due to the way oecore is linked. Ultimately cpu.edl should be removed and debug.edl should only be included for debug builds. But someone has to do the work in OE. I'm also working on supporting preprocessor defines in EDL to allow files to be conditionally included.

Reply:
But yes, I can add a comment